packages/grafana-ui/src/components/Tooltip/Tooltip.tsx

```diff
@@ -67,9 +68,7 @@ export const Tooltip = forwardRef<HTMLElement, TooltipProps>(
     const tooltipId = useId();
 
     const hover = useHover(context, {
-      delay: {
-        close: interactive ? 100 : 0,
-      },
+      ...(interactive ? { handleClose: safePolygon() } : {}),
```

Comment:
_really_ minor nit
```suggestion
      handleClose: interactive ? safePolygon() : undefined,
```
marginally more readable and we avoid having to spread an object. i guess if we were gonna have more prop differences when interactive, the original version would make more sense. but i'm not sure we are 😅

Reply:
No problem, makes sense, I'll make that change now.